charts/pega/Makefile

```diff
@@ -0,0 +1,31 @@
+default: secrets
```

Comment:
When is this file invoked? Can this file be put within HZ subcharts?

Reply:
This file will be invoked manually, please refer: https://github.com/pegasystems/pega-helm-charts/pull/794/files#diff-346646b74f5caaa4e6ffacf9785c6718623b85624c4ad9bd81b6b4496b4d4eb2R1282
This file will generate common certificate for clustering-service and platform, so we are putting it in pega chart